frontend/src/common/models/formik/forgotPasswordSchema.ts

```diff
@@ -0,0 +1,10 @@
+import * as Yup from 'yup';
+
+const forgotPasswordSchema = Yup.object({
+  email: Yup
+    .string()
+    .email('You entered incorrectly email')
+    .required('This field is required.')
```

Comment:
we already have the same validation logic for email in `validationSchemas.ts` file, you can just import it

Reply:
Solved